ethpm_cli/scraper.py

```diff
@@ -198,14 +242,23 @@ def process_entries(
 ) -> Iterable[Tuple[str, str]]:
     for entry in all_entries:
         if entry["address"] == address:
+            logger.info(
+                "<Package %s==%s> released on registry @ %s.\n" "Manifest URI: %s\n",
```

Comment:
*nitpick*.  Why the newline in the middle?

Reply:
Just aesthetics. The entire string"<Package .... URI: %s\n." just seemed a little long & too much data for one line. But I don't have strong feelings about it. Do you think it's better off as a one-liner?

Reply:
I'd be inclined to stick it on one line.  Lets you easily do `wc -l` on the output to count how many packages were found?  I'm sure there are good reasons for either though.